apps/communications/contacts/test/marionette/form_test.js

```diff
@@ -262,4 +262,25 @@ marionette('Contacts > Form', function() {
       assert.ok(isGalleryButtonPresent());
     });
   }); // Facebook Contacts
+
+  suite('> Input buttons', function() {
+    function assertClear(inputSelector) {
+      var input = client.helper.waitForElement(inputSelector);
+      input.tap();
+      input.sendKeys('1234');
+      client.findElement(inputSelector + ' + button[type="reset"]').tap();
+      assert.strictEqual(input.getAttribute('value'), '');
+    }
+
+    test('Clear buttons work as expected', function() {
+      contactsData.createMozContact(contactData);
+      editFirstContact();
+
+      assertClear(selectors.formGivenName);
```

Comment:
nit: instead of reapeating the same sentence various times we can simply perform a loop for each of the fields